protocols/kad/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.44.2 - unreleased
```

Comment:
Sorry, one more thing. Version bump is missing :)

Reply:
Just confirming you are referring to just updating `Cargo.toml` or should this go on and be a minor version bump instead of a patch?